extras/usd/tutorials/endToEnd/tutorial_scripts/add_shadingVariants.py

```diff
@@ -93,6 +93,7 @@ def _AddShadingToBall(stage):
         with shadingVariant.GetVariantEditContext():
             whichBall = variantName.split('_')[-1]
             texPath = os.path.join(texDir, 'ball%s.tex' % whichBall)
+            texPath = texPath.replace('\\', '/')
             # in the current variant, modify the color
```

Comment:
I'm not sure I understand Nick. Are you saying the tutorial script should have a dependency on Tf?